### DISCUSSION TITLE
Help with Fabrication

### DISCUSSION THREAD

**Discussion by IroPlisk** [MrTJP/ProjectRed#1842] [Q&A] [answered]
I couldnt find this anywhere, so i'm asking here: What are the keybinds for the IC workbench? WASD moves, left click places, and arrows shift layers; but how do you rotate the logic gates and other components you place on it? Any and all help is really appreciated, thanks!

**Comment by MrTJP** [comment, marked as answer]
Rotation and other gate configuration is just starting to be implemented (https://github.com/MrTJP/ProjectRed/pull/1840). Once its done, you can configure gates with the Interact tool by zooming into gates and clicking the buttons.

**Reply by IroPlisk** [reply]
ah i see, thanks a ton. Just a last question as i mark this solved: Is such implementation going to come for the 1.19.2 release of the mod? or is it a project with a larger time scope?

**Reply by MrTJP** [reply]
Yup, it's being developed on 1.19. Release 1.20 is alpha, pending dependencies.
